### PR TITLE
Adds Windows support for YOGA

### PR DESCRIPTION
**Pull request by flozz**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -74,7 +74,13 @@ Build and install using the following command::
 Windows
 ~~~~~~~
 
-Building YOGA on Windows is not supported yet...
+The simplest way to use YOGA on Windows is to download the lastest tandalone build here:
```
Comment (clementlevasseur) [inline README.rst:77]: standalone

Comment (jbghoul) [inline README.rst:77]: standalone*

File: winbuild/README-windows-dist.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# YOGA for Windows
+
+> YOGA: Yummy Optimizer for Gorgeous Assets
+
+YOGA is a command-line tool that:
+
+* converts and optimizes images from various format to JPEG and PNG,
+* convertq and optimizeq 3D models from various formats to GLB.
```
Comment (clementlevasseur) [inline winbuild/README-windows-dist.md:8]: converts and optimizes

Comment (jbghoul) [inline winbuild/README-windows-dist.md:8]: converts* and optimizes*

File: winbuild/README-windows-dist.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+Before using YOGA, you must install Microsoft Visual C++ Redistribuable x64 for
+Visual Studio 2019. You can download it from the Microsoft's website
+(`vc_redist.x64.exe`):
+
+* https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads
+
+
+## Install
+
+YOGA do not need any installation. Just unzip the content of this directory and
```
Comment (jbghoul) [inline winbuild/README-windows-dist.md:22]: does not*

File: winbuild/README-windows-dist.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+## Install
+
+YOGA does not need any installation. Just unzip the content of this directory
+and you are ready to go! :)
+
+
+## Usage
+
+YOGA is a CLI (command line) program. To use it, open a terminal in this folder
+and just use one of the following command, depending the terminal you are using.
```
Comment (aardvarksaurus) [inline winbuild/README-windows-dist.md:29]: one of the following commands, depending on the terminal

File: README.rst [hunk 5]
```diff
@@ -74,7 +74,17 @@ Build and install using the following command::
 Windows
 ~~~~~~~
 
-Building YOGA on Windows is not supported yet...
+The simplest way to use YOGA on Windows is to download the lastest standalone build here:
+
+* https://github.com/wanadev/yoga/releases
+
+You will have to install Microsoft Visual C++ Redistribuable for Visual Studio
```
Comment (aardvarksaurus) [inline README.rst:81]: Redistributable

**Pull request merged**